navigator/runtime-compose/src/main/res/values/public.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <public type="id" name="id_that_does_not_exist" />
```

Comment:
is it used?

Reply:
The last time I tried public.xml it didn't work if the file was just empty. So I'm declaring something as public that doesn't exist so that all real resources are considered private.